share/dkg/pedersen/dkg.go

```diff
@@ -78,6 +82,13 @@ type Config struct {
 	// absent) when doing a resharing to avoid a downgrade attack, where a resharing
 	// the number of deals required is less than what it is supposed to be.
 	OldThreshold int
+
+	// Reader holds a user-specified entropy source used to create a random stream
```

Comment:
We should specify what happens when it is set and when it is not set. In particular that when set, dkg still uses the additional crypto reader as well.